### DISCUSSION TITLE
Laravel documentation site unavailable

### DISCUSSION THREAD

**Discussion by andrey-helldar** [laravel/framework#43709] [General] [answered]
Today I encountered the problem of the inaccessibility of the official Laravel website from Russia.

Do I understand correctly that from now on access to the Laravel documentation is closed for Russia, or is this a technical error?

Cause: ERR_CONNECTION_REFUSED

![image](https://user-images.githubusercontent.com/10347617/184661991-1b0b67a5-2a00-4a8b-984e-662671aec713.png)

**Comment by fragkp** [comment, marked as answer]
I would assume (!) it's a problem with cloudfare.

**Reply by andrey-helldar** [reply]
Maybe. My domains work fine through CloudFlare ¯\_(ツ)_/¯

![image](https://user-images.githubusercontent.com/10347617/184717154-6cb68990-29a0-4d46-a725-2bda151edeff.png)

![image](https://user-images.githubusercontent.com/10347617/184717211-7dfe2110-bb63-4e9a-a38a-328fab16f2cd.png)

**Reply by andrey-helldar** [reply]
Probably, there was indeed a technical nuance. Documentation has become available.

![image](https://user-images.githubusercontent.com/10347617/184759321-d56b0cde-1f65-4793-8b34-204f2e71994d.png)
